OvmfPkg/RUNTIME_CONFIG.md

```diff
@@ -0,0 +1,191 @@
+
+```
+qemu-system-x86_64 \
+  -object tls-cipher-suites,id=mysuite0,priority=@SYSTEM \
+  -fw_cfg name=etc/edk2/https/ciphers,gen_id=mysuite0
+```
+
+
+## Network: opt/org.tianocore/IPv4Support
+
+As the name suggests, this enables/disables IPv4 support.  Default:
+enabled.  Usage:
+
+```
+qemu-system-x86_64 -fw_cfg name=opt/org.tianocore/IPv4Support,string=no
+```
+
+## Network: opt/org.tianocore/IPv6Support
+
+Same for IPv6.
+
+
+## Network: opt/org.tianocore/IPv4PXESupport
+
+This enables/disables PXE network boot over IPv4.  This does not
+affect HTTP boot.  Default: enabled.  Usage:
+
+```
+qemu-system-x86_64 -fw_cfg name=opt/org.tianocore/IPv4PXESupport,string=no
+```
+
+
+## Network: opt/org.tianocore/IPv6PXESupport
+
+Same for IPv6.
+
+
+## Network: opt/org.tianocore/VirtioNetSupport
+
+This enables/disables the edk2 driver for virtio net devices.
+Default: enabled.  Usage:
+
+```
+qemu-system-x86_64 -fw_cfg name=opt/org.tianocore/VirtioNetSupport,string=no
+```
+
+
+## Network: opt/org.tianocore/ISCSISupport
+
+This enables/disables the edk2 driver for iSCSI.
+Default: disabled.  Usage:
+
+```
+qemu-system-x86_64 -fw_cfg name=opt/org.tianocore/ISCSISupport,string=yes
+```
+
+
+## Security: opt/ovmf/PcdSetNxForStack
+
+Set stack pages as not executable in page tables.  Usage:
+
+```
+qemu-system-x86_64 -fw_cfg name=opt/ovmf/PcdSetNxForStack,string=yes
+```
+
+
+## Security: opt/org.tianocore/UninstallMemAttrProtocol
+
+Do not provide the `EFI_MEMORY_ATTRIBUTE_PROTOCOL` protocol.  This is
+a workaround for a bug in shim version 15.6.  Usage:
+
+```
+qemu-system-x86_64 -fw_cfg name=opt/org.tianocore/UninstallMemAttrProtocol,string=yes
+```
+
+
+## Platform: opt/org.tianocore/X-Cpuhp-Bugcheck-Override
+
+On some older qemu versions CPU hotplug support was broken.  OVMF
+detects that and will by default log an error message and stop.  When
+setting this option OVMF will continue despite qemu being buggy.  It
+is only safe to set that option if you do not use CPU hotplug.  Usage:
+
+```
+qemu-system-x86_64 -fw_cfg name=opt/org.tianocore/X-Cpuhp-Bugcheck-Override,string=yes
+```
+
+
+## Platform: opt/ovmf/X-PciMmio64Mb
+
+Historically OVMF has by default used 32GB of address space for the
+64-bit PCI MMIO window.  This option can be used to change that size.
+
+Recent OVMF versions have started use an larger MMIO window by
+default, with the actual size depending on the address space size
+supported by the CPU.  With this the default configuration should
+cover alot more use cases and it should be rather rare that this
+option must be used to enlarge the MMIO window.
+
+Usage example (128GB / 131072 MB):
+
+```
+qemu-system-x86_64 -fw_cfg name=opt/ovmf/X-PciMmio64Mb,string=131072
+```
+
+
+## Platform: physical address space bits
+
+OVMF will check the supported physical address space bits (via CPUID)
+when creating the address space layout.  OVMF will use the complete
+available address space and place 64-bit PCI MMIO window near the top

```

Comment:
"place **the** 64-bit PCI MMIO window"

Reply:
Hi @lersek, nice to see you here again ;)
Thanks for proof-reading, fixed the typos.

Reply:
looks good, kudos